ProjectSettings/TagManager.asset

```diff
@@ -20,7 +21,7 @@ TagManager:
   - 
   - Rune
   - Platform
-  - Player
```

Comment:
Could you please add Player back in as a Layer? I think it is needed for the platforms for their "PassengerMask"

Reply:
Added back